infrastructure/main.tf

```diff
@@ -54,6 +54,41 @@ module "fpl-scheduler-db" {
   subscription       = "${var.subscription}"
 }
 
+data "azurerm_key_vault_secret" "fpla_support_email_secret" {
+  name      = "fpla-support-email"
+  vault_uri = "${module.key-vault.key_vault_uri}"
+}
+
+module "fpla-action-group" {
+  source                 = "git@github.com:hmcts/cnp-module-action-group"
+  location               = "global"
+  env                    = "${var.env}"
+  resourcegroup_name     = "fpl-case-service-${var.env}"
+  action_group_name      = "fpla-support"
+  short_name             = "fpla-support"
+  email_receiver_name    = "FPLA Support Mailing List"
+  email_receiver_address = "${data.azurerm_key_vault_secret.fpla_support_email_secret.value}"
+}
+
+module "fpla-performance-alert" {
+  source                     = "git@github.com:hmcts/cnp-module-metric-alert"
+  location                   = "${var.appinsights_location}"
+
+  app_insights_name          = "fpl-case-service-appinsights-${var.env}"
```

Comment:
it is created in line 13, should be same expression or best variable

locals {
  appInsightName = "${var.product}-${var.component}-appinsights-${var.env}"
}

and then ${appInsightName}